packages/components/src/FormGroup/FormGroup.tsx

```diff
@@ -58,7 +58,7 @@ type ItemsProps = {
 /**
  * Helper sub-component for styling the title at the top of the component.
  */
-const Title = ({
+const Title: React.FC<TitleProps> = ({
```

Comment:
This `React.FC<TitleProps>` stuff is required to convince typescript that this is actually a component with a `displayName` property for us to override. Discussion on how to get around this issue in the `Banner` component: https://github.com/chanzuckerberg/edu-design-system/pull/734/files#r759746305

Reply:
When we pass `TitleProps` as a type param, can we remove it from the annotation from the props?

```diff
  className,
-}: TitleProps) => {
+}) => {
```

Reply:
Looks like we can! Update, thanks!